bus-mapping/README.md

```diff
@@ -167,18 +187,13 @@ On that way, we would get something like this for the Memory ops:
 | `0x80` | `0x1d97c6efb` | `Write` | 24   | `73 MSTORE`                              |
 |   -    |               |         |      |                                          |
 | `0xa0` | `0`           | `Write` |      | Init                                     |
-| `0xa0` | `0xcafeb0ba`  | `Write` | 34   | `83 MSTORE`
+| `0xa0` | `0xcafeb0ba`  | `Write` | 34   | `83 MSTORE`                              |
 ```
 
 Where as you see, we group by `memory_address` and then order by
 `global_counter`.
 
-- Iterate over the `ExecutionTrace` itself over
-each `ExecutionStep`'s is formed by and check which Stack/Memory&Storage operations are linked to each step.
+- Iterate over the [`GethExecTrace`](eth_types::GethExecTrace) itself over
+each [`ExecStep`](crate::circuit_input_builder::ExecStep)'s is formed by and check which Stack/Memory&Storage operations are linked to each step.
```

Comment:
nit: not changing line here.

Reply:
I fixed all the line breaks. I also added a GPT round to improve the readability.